src/sidebar/components/Annotation/Annotation.js

```diff
@@ -60,7 +60,7 @@ function Annotation({
 
   return (
     <article
-      className={classnames('Annotation', {
+      className={classnames('Annotation', 'hyp-u-vertical-spacing', {
```

Comment:
There are still references to `Annotation` classes and variants here, which makes it look like there are missing classes in the SASS. From what I can see on master the only style rule that actually existed was equivalent to `hyp-u-vertical-spacing`.

Reply:
I admit this was a punt. I was concerned there could be _some_ class or functionality somewhere in a sub-component that "relied" on being within `.Annotation`. This is probably not the case and I should sort it out. I shall do so.